### ISSUE TITLE
Pseudoexfoliation glaucoma

### ISSUE THREAD

**Issue by Sukramg** [obophenotype/human-phenotype-ontology#7530]
**Preferred term label:**
Pseudoexfoliation glaucoma

**Synonyms**
EXFOLIATION GLAUCOMA; XFG
PSEUDOEXFOLIATION GLAUCOMA; PEXG
PSEUDOEXFOLIATION OF THE LENS
PSEUDOEXFOLIATION SYNDROME; PEXS
Pseudoexfoliative or exfoliative glaucoma (PXFG)

**Definition (free text, please give PubMed ID)**
Secondary open angle glaucomas caused by ocular disease
PMID: 34675001

**Parent term (use hpo.jax.org/app)**
Glaucoma HP:0000501
https://hpo.jax.org/app/browse/term/HP:0000501

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM # 177650
https://www.omim.org/entry/177650?search=%22pseudoexfoliation%20glaucoma%22&highlight=%22pseudoexfoliation%20glaucoma%22
**Your nano-attribution (ORCID)**
0000-0002-6601-4486

**Comment by Sukramg** [comment]
Hello, I have learned that pseudoexfoliation glaucoma in HPO is made up of the terms pseudoexfoliation and glaucoma. 
Therefore, I would like to withdraw this query, but I do not know how.  Thank you.
